java/vector/src/main/java/org/apache/arrow/vector/BaseValueVector.java

```diff
@@ -122,12 +122,17 @@ private static long roundUp8(long size) {
     return ((size + 7) / 8) * 8;
   }
 
+  /* round up bytes for the validity buffer for the given valueCount */
+  private static long roundUp8ForValidityBuffer(int valueCount) {
```

Comment:
you might want to make this package private or public, I think there are other vector classes that have the same roundUp8(getValidityBufferSizeFromCount(..)) logic

Reply:
Make this method public makes sense?

Reply:
> Make this method public makes sense? The least amount of visibility to replace other occurrences of the pattern

Reply:
fixed

Reply:
NM, I couldn't find other methods that use this logic (in which case private makes sense).  sorry for the churn.

Reply:
Alright, fixed now~